tests/test_clauses.py

```diff
@@ -31,3 +32,10 @@ def test_where():
     query, params = render(':v', v=clauses.Where((V('x') == 4) & (V('y').like('xxx'))))
     assert 'WHERE x = $1 AND y LIKE $2' == query
     assert [4, 'xxx'] == params
+
+
+@pytest.mark.parametrize('value,expected', [(0, 0), (False, False), ('', '')])
+def test_falsey_values(value, expected):
+    query, params = render('WHERE :a', a=V('a') == value)
+    assert query == 'WHERE a = $1'
+    assert params == [expected]
```

Comment:
```suggestion
    assert params == [expected]
    assert type(params[0]) == type(expected)
```
Just to make sure params doesn't contain any `SqlBlock`s.